test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -184,12 +184,12 @@ public void ErrorFunction_DoesNotBringDownHost()
             Assert.Equal("loglevel error", traces[4].Message);
 
             // verify most of the logs look correct
-            Assert.EndsWith("Mathew Charles", logs[3]);
```

Comment:
I think the primary issue you're dealing with is that there are now newlines in the log stream, whereaas before there weren't? The test script is just doing a console.log of an object. Are you saying that before this resulted in an unformatted json string, and now it is formatted on multiple lines?

I'd say just change the test script so it writes unformatted json, to facilitate the test code.